CHANGELOG.md

```diff
@@ -4,8 +4,17 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [v5.67.0](https://github.com/opengovsg/FormSG/compare/v5.66.0...v5.67.0)
-
+#### [v5.68.0](https://github.com/opengovsg/FormSG/compare/v5.66.0...v5.68.0)
```

Comment:
This is strange, what happened to 5.67.0? 🤔 Did I mess up something with last week's release? 😰

Reply:
i think cos you didn't push tag @timotheeg

Reply:
recut after you've pushed the tag, so the change log is fixed now